### PR TITLE
[robotraconteur] Update to 1.1.0

### PR DESCRIPTION
**Pull request by johnwason**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: versions/baseline.json [hunk 1]
```diff
@@ -4392,6 +4388,10 @@
       "baseline": "1.5",
       "port-version": 4
     },
+    "libfuse": {
+      "baseline": "3.16.2",
+      "port-version": 0
+    },
```
Comment (jimwang118) [inline versions/baseline.json:4394]:
```suggestion
    "libfuse": {
      "baseline": "3.16.2",
      "port-version": 0
    },
```
Is this port newly added? Why can't I find it in the ports directory?

File: ports/robotraconteur/portfile.cmake [hunk 2]
```diff
@@ -5,8 +5,8 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
 	REPO robotraconteur/robotraconteur
-	REF v1.0.0
-	SHA512 c21dd0af579272c565dd66ca935aababfa3742db524fae66fe82c929561680608e5759ce954f31a8bbcb4ffb7c4e5314f2050b513ccddb8fb49a005c6cfa6d74
+	REF v1.1.0
```
Comment (jimwang118) [inline ports/robotraconteur/portfile.cmake:8]:
```suggestion
	REF "v${VERSION}"
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

File: versions/baseline.json (hunk 1 above)
Comment (johnwason) [reply versions/baseline.json:4394]: I do not know why that part of the file was modified. It was done by the tool. Let me try again.

Comment (johnwason) [reply versions/baseline.json]: This file now only modifies the section for the `robotraconteur` port. There may be a bug in x-add-version that caused this unrelated modification.

File: ports/robotraconteur/portfile.cmake (hunk 2 above)
Comment (johnwason) [reply ports/robotraconteur/portfile.cmake]: Done

File: versions/baseline.json (hunk 1 above)
Comment (dg0yt) [reply versions/baseline.json]: No, x-add-version does it right. But somebody add that port without using x-add-version, and neither CI nor reviewer noticed the glitch. And now it is in almost every PR, and reviewers are surprised... But normally it doesn't cause merge conflicts.

**Review by jimwang118 — APPROVED** [review]

**Comment by johnwason** [comment]
I discovered an issue with version 1.1.0. I will resubmit when repaired.
